dbt_common/context.py

```diff
@@ -26,10 +26,23 @@ def env_secrets(self) -> List[str]:
 _INVOCATION_CONTEXT_VAR: ContextVar[InvocationContext] = ContextVar("DBT_INVOCATION_CONTEXT_VAR")
 
 
+def _reliably_get_invocation_var() -> ContextVar:
```

Comment:
This ensures that the ContextVar is identified by name, which works around some fancy async stuff pytest does which cause this module to be loaded multiple times.

Reply:
Huh, nice discovery with this!